va/va_dec_av1.h

```diff
@@ -1,656 +1,656 @@
-
-            /** \brief read_tx_mode
-             *  value range [0..2]
-             */
-            uint32_t        tx_mode                                     : 2;
-            
-            /* AV1 frame reference mode semantic */
-            uint32_t        reference_select                            : 1;
-
-            uint32_t        reduced_tx_set_used                         : 1;
-
-            uint32_t        skip_mode_present                           : 1;
-            
-            /** \brief Reserved bytes for future use, must be zero */
-            uint32_t        reserved                                    : 20;
-        } bits;
-        uint32_t            value;
-    } mode_control_fields;
-
-    /** \brief CDEF parameters
-     */
-    /*  value range [0..3]  */
-    uint8_t                 cdef_damping_minus_3;
-    /*  value range [0..3]  */
-    uint8_t                 cdef_bits;
-    /*  value range [0..63]  */
-    uint8_t                 cdef_y_strengths[8];
-    /*  value range [0..63]  */
-    uint8_t                 cdef_uv_strengths[8];
-
-    /** \brief loop restoration parameters
-     */
-    union
-    {
-        struct
-        {
-            uint16_t        yframe_restoration_type                     : 2;
-            uint16_t        cbframe_restoration_type                    : 2;
-            uint16_t        crframe_restoration_type                    : 2;
-            uint16_t        lr_unit_shift                               : 2;
-            uint16_t        lr_uv_shift                                 : 1;
-            
-            /** \brief Reserved bytes for future use, must be zero */
-            uint16_t        reserved                                    : 7;
-        } bits;
-        uint16_t            value;
-    } loop_restoration_fields;
-
-    /** \brief global motion 
-     */
-    VAWarpedMotionParamsAV1 wm[7];
-
-    /**@}*/
-
-    /** \brief Reserved bytes for future use, must be zero */
-    uint32_t                va_reserved[VA_PADDING_MEDIUM];
-} VADecPictureParameterBufferAV1;
-
-
-/**
- * \brief AV1 Slice Parameter Buffer Structure
- *
- * This structure conveys parameters related to bit stream data and should be
- * sent once per tile.
- *
- * It uses the name VASliceParameterBufferAV1 to be consistent with other codec,
- * but actually means VATileParameterBufferAV1.
- *
- * Slice data buffer of VASliceDataBufferType is used
- * to send the bitstream.
- *
- * Please note that host decoder is responsible to parse out the
- * per tile information. And the bit stream in sent to driver in per
- * tile granularity.
- */
-typedef struct _VASliceParameterBufferAV1
-{
-    /**@{*/
-    /** \brief The byte count of current tile in the bitstream buffer,
-     *  starting from first byte of the buffer.
-     *  It uses the name slice_data_size to be consistent with other codec,
-     *  but actually means tile_data_size.
-     */
-    uint32_t                Slice_data_size;

```

Comment:
I realize this isn't part of your change, but using uppercase on this member/field name (and others below) don't follow the rest of the vaapi coding style, afaict.  Can you address this, too?  Sorry I did not get to review the original patchset.

Reply:
If you think it's allowed, then I'm ok with it.

Reply:
@uartie I changed it, please help to check it

Reply:
is there other violation? the header file just be merged about 10 hour, so we still have chance to change the variable name, OR, we will have backward compatible issue.